src/main/java/com/cse/ngsa/app/utils/ParameterValidator.java

```diff
@@ -139,4 +140,18 @@ public Boolean isValidRating(String rating) {
     return true;
   }
 
+  /** isValidRating. */
+  public Boolean isValidBenchmarkSize(String benchmarkSizeStr) {
+    if (!StringUtils.isEmpty(benchmarkSizeStr)) {
+      try {
+        int benchmarkSize = Integer.parseInt(benchmarkSizeStr);
+        if (benchmarkSize < 0 || benchmarkSize > BenchmarkController.MaxBenchStrSize) {
```

Comment:
I think this ` BenchmarkController.MaxBenchStrSize` is why you referenced the calling class in this class.  Instead, change in the method input values to accept the max size for comparison or update a config property that may be accessed from here.  My preference is add it as a method parameter here and pass it from the calling function.  If we ever want to put it in properties we can still leave it as described, because we can pass the property value from the calling controller method.